src/main/java/no/unit/nva/datacite/Config.java

```diff
@@ -0,0 +1,62 @@
+package no.unit.nva.datacite;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class Config {
+
+    public static final String ERROR_MISSING_ENVIRONMENT_VARIABLES = "Missing environment variables";
+
+    public static final String ENVIRONMENT_NAME_CORS_ALLOW_ORIGIN_HEADER = "ALLOWED_ORIGIN";
+    public static final String ENVIRONMENT_NAME_DATACITE_MDS_CONFIGS = "DATACITE_MDS_CONFIGS";
+
+    private String corsHeader;
+    private String dataCiteMdsConfigs;
+
+
+    private Config() {
```

Comment:
You should add nva-commons as a dependency with Jitpack: https://github.com/BIBSYSDEV/nva-commons

This is our new micro-framework for common code such as reading environment variables.